lib/conjur/debify/action/publish.rb

```diff
@@ -79,12 +82,12 @@ def publish(options)
         begin
           container.tap(&:start).streaming_logs(follow: true, stdout: true, stderr: true) { |stream, chunk| puts "#{chunk}" }
           status = container.wait
-          raise "Failed to publish #{package_name}" unless status['StatusCode'] == 0
+          raise "Failed to publish package" unless status['StatusCode'] == 0
```

Comment:
Why not include the package name in the message?

Reply:
I removed this because I saw in the error that `package_name` was undefined in this scope. That lead to an even more cryptic error message. Maybe there's a better way to handle this?

Reply:
Sorry, I guess I didn't expand the diff to see the full file. `package_name` is defined in the caller. Looks like I screwed this up when I moved the code around.